modules/virt-configuring-disk-sharing-lun.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-disk-sharing-lun_{context}"]
+= Configuring disk sharing by using LUN
+
+You can configure a LUN-backed virtual machine disk to be shared among multiple virtual machines by enabling SCSI persistent reservation. Enabling the shared option allows you to use advanced SCSI commands, such as those required for a Windows Failover Clustering implementation, against the underlying storage. Any disk to be shared must be in block mode.
+
+A disk of type `LUN` exposes the volume as a LUN device to the VM. This allows the VM to execute arbitrary iSCSI command passthrough on the disk.
+
+You reserve a LUN through the SCSI Persistent Reserve options to protect data on the VM from outside access. To enable the reservation, you configure the feature gate option. You then activate the `PersistentReservation` option on the LUN disk to  issue SCSI device-specific input and output controls (IOCTLs) that the VM requires.
+
```

Comment:
LUN disk to  issue SCSI  >>> remove extra space between 'to' and 'issue'

Reply:
Okay, thanks